### ISSUE TITLE
Issue with model --force

### ISSUE THREAD

**Issue by SimBioT1954** [phalcon/phalcon-devtools#1000]
Hello!
 @sergeyklay 

https://forum.phalconphp.com/discussion/15747/phalcon-devtools-issue

**Relies.php**
```
<?php

namespace MyProject\Models;

class Replies extends BaseModel
{
```

With command **phalcon model replies --namespace=MyProject\Models --force**

I'm getting an error:

```
PHP Fatal error:  Class 'MyProject\Models\BaseModel' not found in /var/www/project/app/models/Replies.php on line 5
```

So I'm not able to update my models!

It happens to me on localhost Windows and Ubuntu 16.04

I know this error occured only if existing model extends BaseModel.

Also I've noticed that on Ubuntu command like **--namespace=MyProject\Models** will create model with namespace without backslash (**MyProjectModels**). 

Nginx, PHP 7.0.15-0ubuntu0.16.04.4 (cli) ( NTS )

**Comment by limingxinleo** [comment]
you can command like --namespace=MyProject\\Models

**Comment by limingxinleo** [comment]
@dreamsxin 您好，又打扰到您，我这里也存在这个问题

**Comment by limingxinleo** [comment]
@SimBioT1954
hello, you can change your config.ini .
`
[devtools]
loader         = "app/models/Model.php"
`
[my phalcon project](https://github.com/limingxinleo/simple-subcontrollers.phalcon)

**Comment by dreamsxin** [comment]
不能加载？@limingxinleo

**Comment by limingxinleo** [comment]
可以加载了，但是需要在config.ini里主动把文件写进来。@dreamsxin

**Comment by sergeyklay** [comment]
> Also I've noticed that on Ubuntu command like --namespace=MyProject\Models will create model with namespace without backslash (MyProjectModels).

This is fixed in the 3.2.x branch.

**Comment by sergeyklay** [comment]
Regarding not found model see: https://github.com/phalcon/phalcon-devtools/issues/571
